flask_lastuser/__init__.py

```diff
@@ -115,49 +115,60 @@ def make_userinfo(self, user):
     def load_user_userinfo(self, userinfo, access_token, update=False):
         raise NotImplementedError("Not implemented in the base class")
 
-    def token_auth(self, resource='*', header_only=False):
+    def get_token(self, header_only):
+        token = None
+
         if 'Authorization' in request.headers:
-            token_match = auth_bearer_re.search(request.headers['Authorization'])
+            token_match = auth_bearer_re.search(
+                request.headers['Authorization'])
             if token_match:
                 token = token_match.group(1)
             else:
                 # Unrecognized Authorization header
-                raise LastuserTokenAuthException(u"A Bearer token is required in the Authorization header.")
+                raise LastuserTokenAuthException(
+                    u"A Bearer token is required in the Authorization header.")
             if 'access_token' in request.values:
-                raise LastuserTokenAuthException(u"Access token specified in both header and body.")
+                raise LastuserTokenAuthException(
+                    u"Access token specified in both header and body.")
```

Comment:
Technically both are in the header, so this message is incorrect. Change it to:

`Access token specified in both Authorization header and query or form.`

Reply:
Done.